### PR TITLE
Update group_rules.yaml to have google related service account by default

### PR DESCRIPTION
**Pull request by BrunoReboul**
update version of group scanner default yaml file to have in the default service accounts

### CODE REVIEW

File: rules/group_rules.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@
       # Must use quotes if the condition starts with @ symbol.
     - member_email: "@{DOMAIN}"
     - member_email: "@gmail.com"
+    # Service Accounts
```
Comment (blueandgold) [inline rules/group_rules.yaml]: Can we make this comment more clear and descriptive of what the `gserviceaccount.com` are?  Such as:
```
GCP service accounts
```

File: rules/group_rules.yaml [hunk 2]
```diff
@@ -19,6 +19,10 @@
...
+    - member_email: "gserviceaccount.com"
```
Comment (blueandgold) [inline rules/group_rules.yaml]: Can we make this opt-in by default, i.e. by commenting this out?  Then if people want it, then can uncomment it.
```
# - member_email: "gserviceaccount.com"
```

File: rules/group_rules.yaml [hunk 3]
```diff
@@ -19,6 +19,10 @@
...
+    # Big Query Transfer Service
+    - member_email: "@bqdts.google.baggins"
```
Comment (blueandgold) [inline rules/group_rules.yaml]: ditto on making this opt-in

**Comment by BrunoReboul** [comment]
Thanks you for the feedback.
I have make the 3 changes

**Pull request merged**